pkg_config.10.xml

```diff
@@ -591,7 +591,7 @@
 		<build_pbi>
 			<port>net/vnstat</port>
 		</build_pbi>
-		<version>1.11_6,2</version>
+		<version>1.11_6,3</version>
```

Comment:
The version number will also need updating in the other older pkg_config etc files to match.